hnn_core/network.py

```diff
@@ -52,20 +55,20 @@ def _create_cell_coords(n_pyr_x, n_pyr_y, zdiff=1307.4):
     pos_dict = dict()
 
     # PYRAMIDAL CELLS
-    xxrange = np.arange(n_pyr_x)
-    yyrange = np.arange(n_pyr_y)
+    xxrange = np.arange(n_pyr_x) * inplane_distance
+    yyrange = np.arange(n_pyr_y) * inplane_distance
 
     pos_dict['L5_pyramidal'] = [
         pos for pos in it.product(xxrange, yyrange, [0])]
     pos_dict['L2_pyramidal'] = [
         pos for pos in it.product(xxrange, yyrange, [zdiff])]
 
     # BASKET CELLS
-    xzero = np.arange(0, n_pyr_x, 3)
-    xone = np.arange(1, n_pyr_x, 3)
+    xzero = np.arange(0, n_pyr_x, 3) * inplane_distance
+    xone = np.arange(1, n_pyr_x, 3) * inplane_distance
```

Comment:
need a couple of simple tests to check that the inplane distance is indeed what's claimed